src/components/Root/Root.jsx

```diff
@@ -0,0 +1,17 @@
+// import { Outlet } from "react-router-dom";
+import Header from "../Header/Header";
+import SiteNav from "../SiteNav/SiteNav";
+// import MainContent from '../MainContent/MainContent';
```

Comment:
The two comments in this file can probably. be removed

Reply:
Thanks Pete, All comments removed!